cpp/CMakeLists.txt

```diff
@@ -163,12 +163,16 @@ if(NOT BUILD_SHARED_LIBS)
   include("${rapids-cmake-dir}/export/find_package_file.cmake")
   list(APPEND METADATA_KINDS BUILD INSTALL)
   foreach(METADATA_KIND IN LISTS METADATA_KINDS)
-    rapids_export_find_package_file(
-      ${METADATA_KIND} "${CUDF_SOURCE_DIR}/cmake/Modules/FindcuFile.cmake" cudf-exports
-    )
+    if(TARGET cufile::cuFile_interface)
```

Comment:
Maybe just do this check once outside the for loop? Then you could also reuse the variable in the `target_link_libraries` below and only need to define the cuFile target once (especially in case the name changes).

Reply:
I have refactored this logic to remove the conditonal inside the tight loop. Does it look good to you know?

Reply:
Yup this looks good now, thanks!